lib/load.php

```diff
@@ -12,18 +12,34 @@
 // These files only need to be loaded if within a rest server instance
 // which this class will exist if that is the case.
 if ( class_exists( 'WP_REST_Controller' ) ) {
-	require dirname( __FILE__ ) . '/class-wp-rest-blocks-controller.php';
-	require dirname( __FILE__ ) . '/class-wp-rest-autosaves-controller.php';
-	require dirname( __FILE__ ) . '/class-wp-rest-block-renderer-controller.php';
-	require dirname( __FILE__ ) . '/class-wp-rest-search-controller.php';
-	require dirname( __FILE__ ) . '/class-wp-rest-search-handler.php';
-	require dirname( __FILE__ ) . '/class-wp-rest-post-search-handler.php';
+	if ( ! class_exists( 'WP_REST_Blocks_Controller' ) ) {
```

Comment:
I'm curious if there's a reason why we use these class checks, rather than `require_once` for every class?

Reply:
@kwight `require_once` works based on file _path_, not the class defined within the file. Because WordPress core's paths are different, you'd end up with fatals if you used `require_once`.